src/utils/components/SSHSecretSection/SSHSecretSection.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Dispatch, FC, SetStateAction, useEffect, useState } from 'react';
+
+import { modelToGroupVersionKind, SecretModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1Secret } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import SecretSelectionRadioGroup from '@kubevirt-utils/components/SSHSecretSection/utils/components/SecretSelectionRadioGroup';
+import SelectSecret from '@kubevirt-utils/components/SSHSecretSection/utils/components/SelectSecret';
+import SSHKeyUpload from '@kubevirt-utils/components/SSHSecretSection/utils/components/SSHKeyUpload/SSHKeyUpload';
+import {
+  SecretSelectionOption,
+  SSHSecretDetails,
+} from '@kubevirt-utils/components/SSHSecretSection/utils/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk-internal';
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import './SSHSecretSection.scss';
+
+type SSHSecretSectionProps = {
+  sshSecretDetails: SSHSecretDetails;
+  setSSHSecretDetails: Dispatch<SetStateAction<SSHSecretDetails>>;
+};
+
+const SSHSecretSection: FC<SSHSecretSectionProps> = ({ sshSecretDetails, setSSHSecretDetails }) => {
+  const [activeNamespace] = useActiveNamespace();
+  const [secretSelectionOption, setSecretSelectionOption] = useState<SecretSelectionOption>(
+    SecretSelectionOption.none,
+  );
+
+  const secretsResourceData = useK8sWatchResource<IoK8sApiCoreV1Secret[]>(
+    activeNamespace
+      ? {
```

Comment:
Does && works as well it it must be null?

Reply:
Done!